src/surfing_penguin/routes.py

```diff
@@ -1,10 +1,10 @@
 """routes.py: Each function in this file indicates a web page (HTML page)."""
 
-from surfing_penguin import surfing_penguin, api
+from src.surfing_penguin import surfing_penguin, api
 from flask import render_template, flash, redirect, url_for
 from flask_restplus import Resource, fields
-from surfing_penguin.forms import LoginForm
-from surfing_penguin.db_interface import Qstnr, UserFunctions
+from src.surfing_penguin.forms import LoginForm
+from src.surfing_penguin.db_interface import Qstnr, UserFunctions
 
```

Comment:
Theses lines would make the original method  (`export FLASK_APP=surfing_penguin.py
flask run`) disable. Please add '\_\_init\_\_.py' under src/ directory to make src a moudle and change readme instruction to `export FLASK_APP=src/surfing_penguin.py
flask run`